add arc_challenge_mt

jonabur opened the pull request:
This PR adds tasks for machine-translated versions of arc challenge for 11 languages.  We will also be adding more languages in the future.

lintangsutawika commented:
Is there any article or paper about this effort? If not, could you make a readme to discuss more about this?

jonabur commented:
I've added a simple readme and added one more eval for icelandic.  We hope to have more information to share once we've done more analysis with these datasets.

jonabur commented:
hi, ping :)

lintangsutawika commented:
Hi, could you expand more about the motivation of this task and the details of the MT process. Typically this is referred to in a paper/blogpost along with certain performance scores that people can also refer to.

haileyschoelkopf reviewed (approved):
Am good with merging this as is personally!
